src/Robot.elm

```diff
@@ -1,4 +1,4 @@
-module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+module Robot exposing (..)

```

Comment:
Don't expose everything. Only `Model`, `Msg`, `init`, `subscriptions`, `update` and `view` should be visible outside. Notice that these are also the only ones you use in `Main`.
Coincidence?
I think not!